### PR TITLE
Display check mark in tray menu to show liked/disliked

### PR DESCRIPTION
**Pull request by donth77**
This is my attempt at implementing the feature request in [#2988](https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/issues/2988).

In the tray menu, a check mark character "✓"(U+2713) is displayed in the "Thumbs Up"  label if the currently playing song is liked and in the "Thumbs Down" label if the song is disliked.

thumbsup

Checks won't pass right away due to failures in master branch.

### CODE REVIEW

**Comment by ctrlrsf** [comment]
LGTM.

**Comment by MarshallOfSound** [comment]
Electron menu items have a built in `checked` state, can you use that instead of this ascii character?

**Comment by donth77** [comment]
@MarshallOfSound I'll look into doing this, it seems that checked MenuItems in electron actually use the same exact character

**Comment by donth77** [comment]
@MarshallOfSound The MenuItems in the tray use the `checked` state to display the check mark. Clicking on the menu items now toggles the liked/disliked state.
capture

File: src/main/features/core/tray.js [hunk 1]
```diff
@@ -105,12 +106,16 @@ const setContextMenu = (track) => {
     },
     { type: 'separator' },
     {
+      type: 'checkbox',
+      checked: PlaybackAPI.getRating().liked,
       label: TranslationProvider.query('playback-label-thumbs-up'),
-      click: () => Emitter.sendToGooglePlayMusic('playback:thumbsUp'),
+      click: () => Emitter.sendToGooglePlayMusic('playback:toggleThumbsUp'),
```
Comment (MarshallOfSound) [inline src/main/features/core/tray.js]: Why did you change this?

Comment (donth77) [reply src/main/features/core/tray.js]: When the menu item is in the checked state and selected, it becomes unchecked and toggles the thumbs up state.

**Comment by MarshallOfSound** [comment]
Superceded by #3321
